environment.yml

```diff
@@ -0,0 +1,22 @@
+name: test-environment
+channels:
+- mosdef
+- bioconda
+- omnia
+- glotzer
+- janschulz
+- defaults
+dependencies:
+- numpy
```

Comment:
We don't need all of these in here. Just use the minimum required (mdtraj and hoomd?)